.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "apps/c/redis/redis"]
```

Comment:
Download redis source directly (like SQLite and iPerf), avoid using the submodule